interfaces/matlab_experimental/Reactors/Wall.m

```diff
@@ -0,0 +1,216 @@
+    % the acceleration or displacement, that is specified. The wall
+    % velocity is computed from:
+    %
+    %    v = K(P_left - P_right) + v_0(t),
+    %
+    % where K is a non-negative constant, and v_0 is a specified
+    % function of time. The velocity is positive if the wall is
+    % moving to the right.
+    %
+    % The heat flux through the wall is computed from:
+    %
+    %    q = U(T_left - T_right) + q_0(t),
+    %
+    % where U is the overall heat transfer coefficient for
+    % conduction/convection, and q_0 is a specified function of
+    % time. The heat flux is positive when heat flows from the
+    % reactor on the left to the reactor on the right.
+    %
+    % Note: the Wall class constructor only assign default values
+    % to various properties. The user could specify those properties
+    % after initial construction by using the various methods of
+    % the 'Wall' class.
+    %
+
+    properties (SetAccess = immutable)
+        id
+        type
+    end
+
+    properties (SetAccess = protected)
+        left % Reactor on the left.
+        right % Reactor on the right.
+    end
+
+    properties (SetAccess = public)
+        area % Area of the wall in m^2.
+    end
+
+    methods
+        %% Wall Class Constructor
+
+        function x = Wall()
+            checklib;
+
+            % At the moment, only one wall type is implemented
+            typ = 'Wall';
+
+            x.type = char(typ);
+            x.id = callct('wall_new', x.type);
+
+            % Set default values.
+            x.left = -1;
+            x.right = -1;
+            x.area = 1.0;
+            x.setExpansionRateCoeff(0.0);
+            x.setHeatTransferCoeff(0.0);
+
+        end
+
+        %% Wall Class Destructor
+
+        function delete(w)
+            % Clear the Wall object from the memory.
+
+            callct('wall_del', w.id);
+        end
+
+        %% Wall Class Utility Methods
+
+        function install(w, l, r)
+            % Install a wall between two reactors.
+            %
+            % w.install(l, r)
+            %
+            % :param l:
+            %    Instance of class 'Reactor' to be used as the bulk phase
+            %    on the left side of the wall.
+            % :param r:
+            %    Instance of class 'Reactor' to be used as the bulk phase
+            %    on the right side of the wall.
+
+            w.left = l;
+            w.right = r;
+            callct('wall_install', w.id, l.id, r.id);
+        end
+
+        function ok = ready(w)
+            % Check whether a wall is ready.
+            %
+            % ok = w.ready
+            %
+
+            ok = callct('wall_ready', w.id);
+        end
+
+        %% ReactorNet get methods
+
+        function a = get.area(w)
+            a = callct('wall_area', w.id);
+        end
+
+        function q = qdot(w, t)
+            % Total heat transfer through a wall at a given time t.
+
+            q = callct('wall_Q', w.id, t);
+        end
```

Comment:
I just checked how `wall_Q` is calculated and it is indeed a heat transfer rate. The original matlab interface docstring incorrectly interpreted it as heat. I think I should just change the name to heatFlux to avoid confusion.

Reply:
> As an aside: it's not clear to me why this needs time - the time should be handled internally by the reactor net simulation; for the time being this is likely a limitation of `clib`, but it should be addressed eventually (not in this PR).

Just a thought: I could assign the `ReactorNet` ID to the `Wall` object so it uses `ReactorNet.time` to evaluate `qdot` and `vdot`. 

For `FlowDevice`, the time-dependent mass flow rate is automatically updated whenever the state of the `ReactorNet` object is updated. There is currently no function that does that for `Wall`.